sdk/monitor/monitor-opentelemetry/src/traces/sampler.ts

```diff
@@ -65,7 +65,7 @@ export class ApplicationInsightsSampler implements Sampler {
     }
     // Add sample rate as span attribute
     attributes = attributes || {};
-    attributes["_MS.sampleRate"] = this._sampleRate;
+    attributes["microsoft.sample_rate"] = this._sampleRate;
```

Comment:
We should only add it if the value is not 100, is mentioned twice in the spec update.

Reply:
I would assume we don't want to send sampleRate = 100 to Breeze and that is why is mentioned there? please confirm.

Reply:
Yeah, it looks like we shouldn't set this if the sample rate is 100 such that ingestion can possibly apply its own sampling. I'll modify this PR.

Reply:
This logic also already exists in the exporter sampler, looks like it just didn't make it to the distro's sampler for trace based logs.

Reply:
The attribute should not be added here anyways, is good we are not sending 100 sample rate in exporter

Reply:
Can you elaborate on what you mean by "the attribute should not be added here"? Are you saying we shouldn't be adding it at all in the distro sampler?